src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/algebras/CharSetSolver.cs

```diff
@@ -97,7 +58,9 @@ public BDD MkCharConstraint(char c, bool ignoreCase = false)
         /// </summary>
         public BDD MkCharSetFromRange(char m, char n)
         {
-            return MkSetFromRange((uint)m, (uint)n, _bw-1);
+            if (m == n)
+                return MkCharConstraint(m);
+            return MkSetFromRange((uint)m, (uint)n, 15);
```

Comment:
Are these changes from `_bw-1` to 15 related to the culture support?

Reply:
Not really, more part of cleanup, I eliminated the _bw field. It is always 16. One could define the constant maxcharbit somewhere as 15  instead of using 15 explicitly.